sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_async.py

```diff
@@ -694,38 +678,6 @@ async def test_multiple_pages_of_results_returned_successfully(self, client):
         assert all([action_result for action_result in recognize_linked_entities_results if len(action_result.document_results) == len(docs)])
         assert all([action_result for action_result in analyze_sentiment_results if len(action_result.document_results) == len(docs)])
 
-    @pytest.mark.skip("Throws 400 on POST: (InvalidRequest) Job task parameter value bad is not supported for model-version "
-                 "parameter for job task type NamedEntityRecognition. Supported values latest,2019-10-01,2020-07-01")
-    @GlobalTextAnalyticsAccountPreparer()
-    @TextAnalyticsClientPreparer()
-    async def test_multiple_pages_of_results_with_errors_returned_successfully(self, client):
```

Comment:
Don't think this test makes sense anymore now that it will always throw at POST. let me know if you think differently

Reply:
agree